packages/filecoin-wallet-provider/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "1.0.0-beta.4",
   "description": "a javascript package to send filecoin to addresses",
   "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "scripts": {
-    "build": "tsc",
+    "build": "rimraf dist/ && tsc --project tsconfig.json",
```

Comment:
I'm cool using rimraf for cleaning dist, but let's do this for all the packages

Reply:
Okay.